src/__tests__/CompareResults/fetchCompareResults.test.tsx

```diff
@@ -6,17 +6,22 @@ import useProtocolTheme from '../../theme/protocolTheme';
 import getTestData from '../utils/fixtures';
 import { renderWithRouter, store } from '../utils/setupTests';
 import { renderHook } from '../utils/test-utils';
-import { screen } from '../utils/test-utils';
+import { screen, act } from '../utils/test-utils';
+
+// function renderWithRoute(component: ReactElement) {
+//   return renderWithRouter(component, {
+//     route:
+//       '/compare-results/?revs=6089e7f0fa57a29c6d080f135f65e146c34457d8,1d5eb1343cc87a9be3dfe4b884822506ffdda7d3&repos=mozilla-central,mozilla-central&framework=1',
+//   });
 
```

Comment:
Can't pinpoint why the snapshot of this route is rendering the Results View page incorrectly in the snapshot.

Reply:
@beatrice-acasandrei, @esanuandra , @julienw I need help figuring why this test won't render the selected revisions component. I've already tried supplying test data to the setSelectedRevisions reducer to mock the selected revisions but for some reason it still won't render the selected revisions component. This route works on dev but this test can't render the selected revisions. This might be a good thing to look over together.

Reply:
Yeah, you expect a RevisionsList but you get a CompareResultsItem in this case. I wonder if you're using the right fixtures?

Reply:
Yes, that was it. Thank you! T-T